.github/workflows/main.yml

```diff
@@ -29,20 +29,14 @@ jobs:
       with:
         fetch-depth: 0
 
-    - name: Build
+    - name: Build default.xml
```

Comment:
Shall we use `matrix` to do this feature?

Reply:
Sure, `matrix` is better in this case.